src/Bicep.LangServer/Server.cs

```diff
@@ -79,13 +88,14 @@ public async Task RunAsync(CancellationToken cancellationToken)
             await server.WaitForExit;
         }
 
-        private static void RegisterServices(CreationOptions creationOptions, IServiceCollection services)
+        private void RegisterServices(CreationOptions creationOptions, IServiceCollection services)
```

Comment:
why remove static?

Reply:
That was unintentional. Reverted it here:
https://github.com/Azure/bicep/commit/defae18da5e33a60b31cb9421da03f639ae9b375

Thanks!